rllib/algorithms/algorithm_config.py

```diff
@@ -888,6 +888,18 @@ def validate(self) -> None:
                 "False)."
             )
 
+        if (
```

Comment:
can we comment here that this is a short-term workaround?
this is a pretty weird requirement by itself.

Reply:
sure

Reply:
done